org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/SchemaAssociationsPreferenceInitializer.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.wildwebdeveloper;
+
+import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer;
+import org.eclipse.jface.preference.IPreferenceStore;
+
+import com.google.gson.JsonObject;
+
+public class SchemaAssociationsPreferenceInitializer extends AbstractPreferenceInitializer {
+	private static final IPreferenceStore STORE = Activator.getDefault().getPreferenceStore();
+	public static final String SCHEMA_ASSOCIATIONS_PREFERENCE = "wildwebdeveloper.yaml.schema";
```

Comment:
That may be the right place where to also read the legacy extension point for JSon.